ira-task-2/styles.css

```diff
@@ -0,0 +1,4 @@
+body {
+    padding: 3%;

```

Comment:
a good practice is to not give padding margin font size in percentage either use rem or px